scripts/gas_stats.py

```diff
@@ -744,6 +740,185 @@ def borrowActions(env, withdrawCash, redeem):
     )
     log_gas("batchAction.{}.DepositETHCollateral".format(key), txnCold, txnWarm)
 
+def _enable_cash_group(currencyId, env, accounts, initialCash=50000000e8):
+    env.notional.updateDepositParameters(currencyId, *(nTokenDefaults["Deposit"]))
+    env.notional.updateInitializationParameters(currencyId, *(nTokenDefaults["Initialization"]))
+    env.notional.updateTokenCollateralParameters(currencyId, *(nTokenDefaults["Collateral"]))
+    env.notional.updateIncentiveEmissionRate(currencyId, CurrencyDefaults["incentiveEmissionRate"])
+
+    env.notional.batchBalanceAction(
+        accounts[0],
+        [
+            get_balance_action(
+                currencyId, "DepositAssetAndMintNToken", depositActionAmount=initialCash
+            )
+        ],
+        {"from": accounts[0]},
+    )
+    env.notional.initializeMarkets(currencyId, True)
+
+def fcashLiquidateSetup(env):
+    cToken = env.cToken["ETH"]
+    cToken.mint({"from": accounts[0], "value": 10000e18})
+    cToken.approve(env.notional.address, 2 ** 255, {"from": accounts[0]})
+
+    _enable_cash_group(1, env, accounts, initialCash=40000e8)
+    cashGroup = list(env.notional.getCashGroup(2))
+    # Enable the one year market
+    cashGroup[0] = 3
+    cashGroup[9] = CurrencyDefaults["tokenHaircut"][0:3]
+    cashGroup[10] = CurrencyDefaults["rateScalar"][0:3]
+    env.notional.updateCashGroup(2, cashGroup)
+
+    env.notional.updateDepositParameters(2, [0.4e8, 0.4e8, 0.2e8], [0.8e9, 0.8e9, 0.8e9])
+
+    env.notional.updateInitializationParameters(
+        2, [1.01e9, 1.021e9, 1.07e9], [0.5e9, 0.5e9, 0.5e9]
```

Comment:
These should change to:
`2, [0.01e9, 0.021e9, 0.07e9], [0.5e9, 0.5e9, 0.5e9]`

Reply:
fixed